pkg/server/params/params.go

```
@@ -45,7 +44,9 @@ func ParsePaging(paging string) (limit, offset int) {
 	return
```

why ignore error here? what would happen if there is an error during parsing?

---

Use default values when parsing failed, limit and page must be defined at the same time.

---

I believe atoi will return 0 when there is an error

---

Regular expression `\d+` defines that only numbers are allowed here, i think this error can be ignored.